src/timer/ExamTimerBlock.jsx

```diff
@@ -6,7 +6,7 @@ import CountDownTimer from './CountDownTimer';
 import { ExamStatus, IS_STARTED_STATUS } from '../constants';
 import TimerProvider from './TimerProvider';
 import {
-  Emitter, expireExamAttempt, stopExam, submitExam,
+  Emitter, expireExam, stopExam, submitExam,
```

Comment:
I don't think anything else in the library uses the `expireExamAttempt` function, so I think we should remove the export from `data/index.js` as well.